DSI/globus_gridftp_server_iRODS.cpp

```diff
@@ -1740,7 +1923,7 @@ globus_l_gfs_iRODS_recv(
 
     // start N threads to write to file
     int number_of_irods_write_threads = iRODS_handle->number_of_irods_read_write_threads;
-    irods::thread_pool threads{number_of_irods_write_threads};
+    irods::thread_pool threads{number_of_irods_write_threads-1};
```

Comment:
This is -1 because the main thread is also writing, but separate from the `thread_pool`? Worth adding a comment to that effect?